src/compute/src/server.rs

```diff
@@ -212,8 +213,32 @@ pub async fn compute_node_serve(
         streaming_metrics.clone(),
         config.streaming.clone(),
         async_stack_trace_config,
-        opts.total_memory_bytes,
     ));
+
+    // Spawn LRU Manager that have access to collect memory from batch mgr and stream mgr.
+    let batch_mgr_clone = batch_mgr.clone();
+    let stream_mgr_clone = stream_mgr.clone();
+    let create_lru_manager = move || {
+        let mgr = GlobalMemoryManager::new(
+            opts.total_memory_bytes,
+            config.streaming.barrier_interval_ms,
+            streaming_metrics.clone(),
+        );
+        // Run a background memory monitor
+        tokio::spawn(mgr.clone().run(batch_mgr_clone, stream_mgr_clone));
```

Comment:
It should be a long running task, which trigger the `run` method periodically, but we can refine it later.

Reply:
There is a tick interval in `run` so I guess this is not a big problem?